beluga/include/beluga/sensor/ndt_sensor_model.hpp

```diff
@@ -105,17 +105,28 @@ inline std::vector<NDTCell<NDim, Scalar>> to_cells(
 }
 /// Default neighbor kernel for the 2D NDT sensor model.
 const std::vector<Eigen::Vector2i> kDefaultNeighborKernel2d = {
-    Eigen::Vector2i{-1, -1}, Eigen::Vector2i{-1, -0}, Eigen::Vector2i{-1, 1},
-    Eigen::Vector2i{0, -1},  Eigen::Vector2i{0, 0},   Eigen::Vector2i{0, 1},
-    Eigen::Vector2i{1, -1},  Eigen::Vector2i{1, 0},   Eigen::Vector2i{1, 1},
+    Eigen::Vector2i{-1, -1},  //
+    Eigen::Vector2i{-1, -0},  //
+    Eigen::Vector2i{-1, 1},   //
+    Eigen::Vector2i{0, -1},   //
+    Eigen::Vector2i{0, 0},    //
+    Eigen::Vector2i{0, 1},    //
+    Eigen::Vector2i{1, -1},   //
+    Eigen::Vector2i{1, 0},    //
+    Eigen::Vector2i{1, 1},    //
 };
 
 /// Default neighbor kernel for the 3D NDT sensor model.
 const std::vector<Eigen::Vector3i> kDefaultNeighborKernel3d = {
     // TODO(Ramiro) revisit this kernel when we implement the 3D sensor model and extend this if it's computationally
     // feasible.
```

Comment:
@serraramiro1 :eyes:

Reply:
Still unclear to me without testing this E2E on a node. :(